uber/config.py

```diff
@@ -73,9 +73,19 @@ def make_enum(self, enum_name, section, prices=False):
                 lookup[val] = desc
 
         enum_name = enum_name.upper()
-        setattr(self, enum_name + '_OPTS', opts)
-        setattr(self, enum_name + '_VARS', varnames)
-        setattr(self, enum_name + ('' if enum_name.endswith('S') else 'S'), lookup)
+        # When loading plugin's configs, we want to make sure we don't override existing enums
+        if hasattr(self, enum_name + '_OPTS'):
```

Comment:
@binary1230's suggestion is a good one, but would probably end up being a lot more work than "just" implementing enum-merging here.  We can't modify Sideboard to just always merge in all plugin configs, because a lot of Sideboard plugins (including all of the ones I've written here at work) emphatically don't want that.  Still, we could make it an option.

The real problem is that plugin order matters (e.g. we need ``uber`` to be loaded before ``magprime``), and plugin order is defined... in the config!  So we've got a yo dawg scenario where we need to read from our config to tell us how to parse and merge our config.

This is totally doable, since we can always do something like read from our unmerged and uninterpolated config to figure out whether to merge in plugin config and check for plugin load order, then do the loading/merging, then interpolate and return the finalized config.  But that's probably getting into "maybe more work than it's worth for what we're trying to do here".  Or not, depending on what people's willingness to tackle this kind of stuff is.  (I'm looking forward to having like a month sometime when I'm not juggling panels and MIVS and hotel and whatever else to take on these kinds of capital improvements.)